### ISSUE TITLE
C++ params with namespaces have the namespace as docstring name

### ISSUE THREAD

**Issue by rmatpd** [kkoomen/vim-doge#17]
The following is what was generated from a function with namespaces:

```
/**
 * TODO
 *
 * @param std TODO
 * @return TODO
 */
std::string trimLine(const std::string &line);
```

the "@param std TODO" is incorrect, and should instead be "@param string TODO"

**Comment by kkoomen** [comment]
I need the whole function expression to debug this. Can you provide this?

**Comment by kkoomen** [comment]
It should be `@param line TODO`, right? `string` is the type hint, but it should be the var name.

**Comment by kkoomen** [comment]
This has been fixed in [v1.4.6](https://github.com/kkoomen/vim-doge/tree/v1.4.6).

Feel free to submit any new issues if you experience any unwanted behaviour in the future. Thanks for your contribution.

**Comment by rmatpd** [comment]
You're correct about my incorrect param name.  Thanks for the quick fix!
